src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx

```diff
@@ -17,8 +17,7 @@ import opentelemetryJiraIncidentManagement from 'images/opentelemetry_screenshot
 import opentelemetryJiraNRConfiguration from 'images/opentelemetry_screenshot-full_jira-nr-configuration.webp'
 
 
-
-New Relic offers an integration with Atlassian Jira that enables the incident tab feature within Jira. This feature syncs all your New Relic issue and incident information with Atlassian. This data then appears in Jira on the incidents tab and lets you connect it with Jira issues to improve your outage workflows. The sections cover how you set up this integration and the best practices for usage.
+New Relic offers an integration with Atlassian Jira that syncs your New Relic issue and incident information with Jira. This data then appears in Jira in the incidents feature and lets you connect it with Jira issues to improve your outage workflows. The sections cover how to set up this integration and best practices for using it.
```

Comment:
@nbaenam Maybe add a docs link to the "incidents feature" for reference

Reply:
Good idea, @ally-sassman! Done it!